monitor/tests/test_health.py

```
@@ -0,0 +1,178 @@
+            mock_async.side_effect = Exception("Test error")
+            
+            result = call_health(service)
+            
+            assert result['ok'] is False
+            assert "Wrapper error" in result['error_text']
+    
+    def test_call_health_batch(self):
+        """Test batch health checking."""
+        services = baker.make(Service, _quantity=3)
+        
+        async def run_batch_test():
```

Ruff (0.12.2)

Missing return type annotation for private function `run_batch_test`

Add return type annotation: `None`

(ANN202)